llvmlite/ir/builder.py

```diff
@@ -1016,3 +1030,24 @@ def fma(self, a, b, c):
         """
         Perform the fused multiply-add operation.
         """
+
+    def convert_from_fp16(self, a, to=None, name=''):
+        """
+        Convert from an i16 to the given FP type
+        """
+        if not to:
+            raise TypeError("expected a float return type")
```

Comment:
Is this branch exercised in the tests?

Reply:
I've added another test so all the error messages are covered now